src/migration/ember-addon/steps/update-addon-package-json.js

```diff
@@ -71,7 +71,7 @@ function updateDevDependencies(packageJson, options) {
       ];
 
   packagesToInstall.forEach((packageName) => {
-    const version = decideVersion(packageName, devDependencies);
+    const version = decideVersion(packageName, options);
```

Comment:
**bugfix**

We should pass `options` (to be precise, `augmentedOptions`) because it includes the combined list of dependencies from `dependencies` and `devDependencies`.